packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -16,6 +16,8 @@ import {
   WarningFilled,
 } from '@carbon/icons-react';
 import ListBox, { PropTypes as ListBoxPropTypes } from '../ListBox';
+import ListBoxMenuItem from '../ListBox/ListBoxMenuItem';
+import ListBoxMenu from '../ListBox/ListBoxMenu';
```

Comment:
I think it's possible that this could break tests in downstream consumer's projects that rely on the component name to select nodes/components in test assertions. Is there a way to retain the usage of the namespaced exports (`ListBox.Menu`, etc.) instead of these direct ones?

Reply:
Yeah, I don't recall the reason for changing this.